src/components/CheckoutForm.js

```diff
@@ -77,29 +88,24 @@ class CheckoutForm extends Component {
         </Form>
       );
     }
-
-    if (this.state.success !== "") {
-      paymentStatus = (
-        <Card>
-          <Card.Content header={this.state.success} />
-        </Card>
-      );
-    }
-
-    if (this.state.error !== "") {
-      paymentStatus = (
-        <Card>
-          <Card.Content header={this.state.error} />
-        </Card>
-      );
-    }
     return (
       <Container>
-        {paymentStatus}
         {stripeForm}
       </Container>
     );
   }
 }
 
-export default injectStripe(CheckoutForm);
+const mapDispatchToProps = {
+  dispatchFlash: (message, status) => ({
+    type: "SHOW_FLASH_MESSAGE",
+    payload: { flashMessage: message, status: status }
+  })
+};
+
+export default injectStripe(
```

Comment:
Same here, what does this do?

Reply:
this is a requirement to use stripe in the component

Reply:
Got it.